src/Helper/ChoiceQuestion.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace TheAentMachine\Helper;
+
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * A helper class to easily create choice questions.
+ */
+class ChoiceQuestion extends BaseQuestion
```

Comment:
We should use some kind of validator for help text and default value?

Reply:
Symfony's ChoiceQuestion has its own validator which is pretty good (you can multiselect using commas or spaces as separator)